MixItUp.WPF/Controls/Actions/CounterActionControl.xaml.cs

```diff
@@ -83,9 +84,15 @@ private void CounterActionTypeComboBox_SelectionChanged(object sender, System.Wi
             }
         }
 
-        private void CountersFolderHyperlink_Click(object sender, System.Windows.RoutedEventArgs e)
+        private async void CountersFolderHyperlink_Click(object sender, System.Windows.RoutedEventArgs e)
         {
-            Process.Start(Path.Combine(Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location), CounterAction.CounterFolderName));
+            string counterFolderPath = Path.Combine(Path.GetDirectoryName(typeof(CounterActionControl).Assembly.Location), CounterAction.CounterFolderName);
```

Comment:
Put this as an interface method on IFileService (Just have it typeof(IFileService) for it) as it'd be a good generic one to add. Something like "GetApplicationDirectory()":

https://github.com/SaviorXTanren/mixer-mixitup/blob/master/MixItUp.Base/Services/IFileService.cs

Desktop implementation of that interface:

https://github.com/SaviorXTanren/mixer-mixitup/blob/master/MixItUp.Desktop/Services/WindowsFileService.cs

Reply:
Addressed.